Specs/Widgets/Viewer/ViewerSpec.js

```diff
@@ -1021,4 +1025,165 @@ defineSuite([
+        viewer = createViewer(container);
+
+        var path = './Data/Cesium3DTiles/Tilesets/TilesetOfTilesets';
+        var tileset = new Cesium3DTileSet({
+            url : path
+        });
+
+        // load tileset to test
+        return tileset.readyPromise.then(function() {
+            var promise = viewer.flyTo(tileset);
+            var wasCompleted = false;
+
+            spyOn(viewer.camera, 'flyToBoundingSphere').and.callFake(function(target, options) {
+                wasCompleted = true;
+                options.complete();
+            });
+
+            viewer.render();
+
+            return promise.then(function() {
+                expect(wasCompleted).toEqual(true);
+            });
+        });
+
+    });
+
+    it('flys to when target is Cesium3DTileset and uses default offset when offset not defined in options', function() {
+        viewer = createViewer(container);
+
+        var path = './Data/Cesium3DTiles/Tilesets/TilesetOfTilesets';
+        var tileset = new Cesium3DTileSet({
+            url : path
+        });
+
+        var options = {};
+
+        // load tileset to test
+        return tileset.readyPromise.then(function() {
+            var promise = viewer.flyTo(tileset, options);
+            var wasCompleted = false;
+
+            spyOn(viewer.camera, 'flyToBoundingSphere').and.callFake(function(target, options) {
+                wasCompleted = true;
+                options.complete();
+            });
+
+            viewer.render();
+
+            return promise.then(function() {
+                expect(wasCompleted).toEqual(true);
+            });
+
+        });
+    });
+
+    it('zooms to target when target is Cesium3DTileset and offset is defined', function() {
+        viewer = createViewer(container);
+
+        // stored for movement check
+        var camPos = Cartesian3.clone(viewer.camera.position);
+        var camDir = Cartesian3.clone(viewer.camera.direction);
+
+        var path = './Data/Cesium3DTiles/Tilesets/TilesetOfTilesets';
+        var tileset = new Cesium3DTileSet({
+            url : path
+        });
+
+        //spyOn(viewer.camera, 'viewBoundingSphere');
+
+        // load the tileset then check tests
+        return tileset.readyPromise.then(function() {
+            var boundingSphere = tileset.boundingSphere;
+            var offset = new HeadingPitchRange(0.4, 1.2, 4 * boundingSphere.radius);
+
+            return viewer.zoomTo(tileset, offset).then(function() {
+                // called with proper values
+                //expect(viewer.camera.viewBoundingSphere).toHaveBeenCalledWith(boundingSphere, offset);
+                // moved to new location
+                expect(viewer.camera.position).not.toEqual(camPos);
+                expect(viewer.camera.direction).not.toEqual(camDir);
+            });
+        });
+    });
+
+    it('flys to target when target is Cesium3DTileset and offset is defined', function() {
```

Comment:
I think your tests names can be a little more precise. For instance, this one can be "flyTo flies to Cesium3DTileset with offset".

"flys to when target is Cesium3DTileset and uses default offset when offset not defined in options" -> "flyTo flies to Cesium3DTileset with default offset"